docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -448,6 +448,9 @@ You can set:
 - `spring.cloud.loadbalancer.retry.maxRetriesOnSameServiceInstance` - indicates how many times a request should be retried on the same `ServiceInstance` (counted separately for every selected instance)
 - `spring.cloud.loadbalancer.retry.maxRetriesOnNextServiceInstance` - indicates how many times a request should be retried a newly selected `ServiceInstance`
 - `spring.cloud.loadbalancer.retry.retryableStatusCodes` - the status codes on which to always retry a failed request.
+
+NOTE: For load-balanced retries, by default, we wrap the `ServiceInstanceListSupplier` bean with `RetryAwareServiceInstanceListSupplier` to select a different instance from the one previously chosen, if available. You can disable this behaviour by setting the value of `spring.cloud.loadbalancer.retry.avoidPreviousInstance` to `false`.
```

Comment:
Take the u out of behaviour. (Spring uses American spellings.) My bad for not catching that earlier. I am too used to reading British English.